Adding documentation about Qiskit Transpiler Service limits (#1941)
Qiskit/documentation: docs/guides/qiskit-transpiler-service.mdx

    transpiled_circuit = cloud_transpiler_service.run(circuit)
    ```

## Limits of the Qiskit Transpiler Service

> The following lists the most relevant limits of the service:

Reviewer: ```suggestion
Following are the most relevant limitations of the service:
```

> - The Qiskit Transpiler Service transpiling up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.

Reviewer: ```suggestion
- The Qiskit Transpiler Service can transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.
```

> - The Qiskit Transpiler Service can transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.

Reviewer 2: ```suggestion
- The Qiskit Transpiler Service can transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"`, or `ai="auto"`.
```

- The maximum time allowed to run a transpilation process is 30 minutes.

> - The maximum time to retrieve a transpilation result from the service after the transpilation process ends is 20 minutes. After 20 minutes, your job will be removed.

Reviewer: ```suggestion
- You must retrieve the transpilation result from the service within 20 minutes after the transpilation process ends. After 20 minutes, your job is removed.
```

> - The maximum time to retrieve a transpilation result from the service after the transpilation process ends is 20 mins. After 20 mins your job will be removed.

Reviewer 3: I think the results are removed after `30min` right now.

Author: I double-checked and in the service config I see 1200 for polling max time for results.

> ## Limits of the Qiskit Transpiler Service
> 
> The service has some limits enabled, here we list the most relevant ones:
> 
> - The Qiskit Transpiler Service allows to transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.
> - The maximum time allowed to run a transpilation process is 30 mins.
> - The maximum time to retrieve a transpilation result from the service after the transpilation process ends is 20 mins. After 20 mins your job will be removed.

Reviewer 2: ```suggestion
## Limits of the Qiskit Transpiler Service

The following lists the most relevant limits of the service:

- The Qiskit Transpiler Service transpiling up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.
- The maximum time allowed to run a transpilation process is 30 minutes.
- The maximum time to retrieve a transpilation result from the service after the transpilation process ends is 20 minutes. After 20 minutes, your job will be removed.
```

> - The maximum time a set of circuits can live in the internal queue while waiting to be transpiled is 120 minutes. After that time, if the job has not been transpiled, it will be discarded.

Reviewer: ```suggestion
- The maximum time a set of circuits can live in the internal queue while waiting to be transpiled is 120 minutes. After that time, if the job has not been transpiled, it is discarded.
```

> - The maximum number of qubits has not been determined. The service has been tested up to 900+ qubits.

Reviewer: ```suggestion
- The maximum number of qubits has not been determined. The service has been tested on 900+ qubits.
```

> - The maximum time a set of circuits can live in the internal queues while waiting to be transpiled is 120 mins. After that time, if the job has not been transpiled, it will be discarded.
> - The maximum number of qubits is not really fixed. Though, the service has been only tested up to 900+ qubits.

Reviewer 2: ```suggestion
- The maximum time a set of circuits can live in the internal queue while waiting to be transpiled is 120 minutes. After that time, if the job has not been transpiled, it will be discarded.
- The maximum number of qubits has not been determined. The service has been tested up to 900+ qubits.
```

## Limits of the Qiskit Transpiler Service

The following lists the most relevant limits of the service:

> - The Qiskit Transpiler Service transpiling up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.

Reviewer 2: ```suggestion
- The Qiskit Transpiler Service can transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"` or `ai="auto"`.
```
